domain/src/main/java/org/oppia/android/domain/topic/StoryProgressTestHelper.kt

```diff
@@ -235,8 +235,7 @@ class StoryProgressTestHelper @Inject constructor(
    * Marks full topic progress on the second test topic for a particular profile.
    *
    * @param profileId The profile we are setting topic progress for.
-   * @param timestampOlderThanOneWeek If the timestamp for completing the topic is from more than
-   *     one week ago.
+   * @param timestampOlderThanOneWeek If the timestamp for completing the topic is from more than one week ago.
```

Comment:
nit -- I believe it's convention for the param to be part of the sentence and use a period only if it's more than one sentence, so something like:
`@param profileId the profile we are setting topic progress for`
and
`@param timestampOlderThanOneWeek for if the timestamp for completing the topic is from more than one week ago`

Reply:
Done.